### DISCUSSION TITLE
Cleanout Accounts Cache?

### DISCUSSION THREAD

**Discussion by vsatmydynipnet** [mastodon/mastodon#19973] [Q&A]
Is there a way to cleanout system and cache files from accounts and accounts/headers  in?

./home/mastodon/live/public/system/accounts/headers/000
./home/mastodon/live/public/system/cache/accounts/avatars
./home/mastodon/live/public/system/cache/accounts/headers

?

These contains very old files on my server, so I assume there is a way to cleanout things I have not found till now.

 Thank you for any suggestions

**Comment by quicoto** [comment]
After reading #9567 I don't think there's a built-in way to clean the remote avatars and headers. Which is very surprising and troubling.

**Reply by vsatmydynipnet** [reply]
The question is, if I run:

```
cd /home/mastodon/live && find <directory> -type f -atime +7 -delete

for 
./home/mastodon/live/public/system/accounts/headers/000
./home/mastodon/live/public/system/cache/accounts/avatars
./home/mastodon/live/public/system/cache/accounts/headers
as directory
```

- Do I delete local account images with it?
- If remote ones are cleared, are they refetched if needed?

**Reply by quicoto** [reply]
I have no idea, sorry. but I'd also like to know. Someone needs to try, perhaps on a dev instance 😅

Another crazy idea would be to play it safe and replace all these images by another one, which would be a super small 1x1 pixel. The weight would go down significantly. Granted, you would still have the thousands of files 🤔

**Comment by vsatmydynipnet** [comment]
Sorry, no dev setup available here. I have 

5G    36093 Files     ./home/mastodon/live/public/system/accounts
13G  75601 Files     ./home/mastodon/live/public/system/cache/accounts

I am old fashioned, having an eye on disk usage to also keep the backups as small as possible. I assume this could be cleaned up to 1-3 G

**Reply by quicoto** [reply]
Maybe do a backup of the folders and give it a go?

**Comment by vsatmydynipnet** [comment]
This does not look good for us:

https://github.com/mastodon/mastodon/issues/9567

Some hoping since 4 years for a fix to this one.

**Reply by quicoto** [reply]
@vsatmydynipnet It's no silver bullet but replacing the headers with 1 pixel images has alleviated the issue.

Here are my findings from the past few days:

https://ricard.dev/improving-mastodons-disk-usage/

Let me know your thoughts!

**Comment by vsatmydynipnet** [comment]
I still do hope that this thing gets an official solution by @Gargron because it affects so much small instance operators. Replacing with 1x1 is not what I want to do, this would probably break to much.

**Reply by quicoto** [reply]
Indeed an official solution is ideal 💯 

So far I don't any anything breaking for me. When I access any profile the header loads, same file name, no UI changes, all looks good. It's just a very small image file.
